lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -276,39 +313,45 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
     }
 
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
-        if let nextTextField = textField.superview?.superview?.viewWithTag(textField.tag + 1) as? UITextField {
+        if let nextTextField = textField.superview?.superview?.superview?.viewWithTag(textField.tag + 1) as? UITextField {
```

Comment:
I know there wasn't a substantive change here, but this feels really fragile. I think a better solution might be to either use something like IQKeyboardManager to handle this for you. Or maybe you could keep an array of the textfields in the desired order and get the next `firstResponder` by checking that array. I think there are several good options.

Reply:
I went with the array method. Thank you for catching that. The array option was much cleaner.